hazelcast-sql/src/test/java/com/hazelcast/jet/sql_slow/MapScanMigrationStressTest.java

```diff
@@ -62,13 +63,18 @@ public void before() throws InterruptedException {
         factory = new TestHazelcastFactory();
         instances = new HazelcastInstance[4];
         for (int i = 0; i < instances.length - 1; i++) {
-            instances[i] = factory.newHazelcastInstance(smallInstanceConfig());
+            instances[i] = factory.newHazelcastInstance(createFastRetryConfig());
         }
         SqlTestSupport.createMapping(instances[0], MAP_NAME, Integer.class, Integer.class);
         map = instances[0].getMap(MAP_NAME);
         mutatorException = new AtomicReference<>(null);
     }
 
+    private static Config createFastRetryConfig() {
+        return smallInstanceConfig()
```

Comment:
A comment why this is needed would be beneficial.

Reply:
Done